definitions.md

```diff
@@ -0,0 +1,50 @@
+# Definition of Common Terms
+
+## Infrastructure
+
+- <a name="hub"/>**Hub**:
+    - **Definition**: A kubernetes cluster that is running OCM, typically connected to one or more [spokes](#spoke).
+    - Common alternatives to avoid:
+        - Control plane
+        - Control cluster
+- <a name="spoke"/>**Spoke**:
+    - **Definition**: A Kubernetes cluster connected to a [hub](#hub) where resources can be placed via a manifestwork.
+    - Common alternatives to avoid:
+        - Workload cluster
+        - Cluster
+        - Gateway cluster
+        - Data plane
+        - Managed cluster
+- <a name="workload"/>**Workload**:
+    - **Definition**: A set of resources deployed to one or more [spokes](#spoke) - typically accessed via a [gateway](#gateway) - and designed to complete a task, e.g. an API or web-page.
+    - Common Alternatives to avoid:
+        - Application
+        - API
+        - deployment
+        - backend
+- <a name="gateway"/>**Gateway**:
+    - **Definition**:
+        - On a [spoke](#spoke): A resource designed to facilitate ingress to a [workload](#workload)
+        - On a [hub](#hub): A resource which follows specific mutation and replication rules to be replicated into [spokes](spokes).
+
+## Personas
```

Comment:
I think we should align with the existing personas at https://gateway-api.sigs.k8s.io/#personas,
and supplement them with the Platform Engineer persona, who's concern lies beyond a single cluster

Reply:
this is a great suggestion, thanks Dave, I'll do this.